Description: Upstream changes introduced in version 2:2.0.0~rc2-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 redis (2:2.0.0~rc2-1) experimental; urgency=low
 .
   * New upstream RC release.
 .
 The person named in the Author field signed this changelog entry.
Author: Chris Lamb <lamby@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ redis-2.0.0~rc2/release.h
@@ -0,0 +1,2 @@
+#define REDIS_GIT_SHA1 "57f67f9f"
+#define REDIS_GIT_DIRTY "0"
